apps/src/templates/sectionProgress/sectionProgressRedux.js

```diff
@@ -270,9 +317,11 @@ export const loadScript = (scriptId) => {
     }
 
     dispatch(startLoadingProgress());
-    $.getJSON(`/dashboardapi/script_structure/${scriptId}`, scriptData => {
-      dispatch(addScriptData(scriptId, scriptData));
-    });
+    const scriptRequest = fetch(`/dashboardapi/script_structure/${scriptId}`, {credentials: 'include'})
```

Comment:
Outside the scope of this PR, but this function might look prettier if redone with `async`/`await`.